lang/python/python3/files/python3-package-pip.mk

```diff
@@ -22,10 +22,8 @@ define Py3Package/python3-pip/install
 		$(PKG_BUILD_DIR)/install-pip/usr/lib/python$(PYTHON3_VERSION)/site-packages/pip \
 		$(PKG_BUILD_DIR)/install-pip/usr/lib/python$(PYTHON3_VERSION)/site-packages/pip-$(PYTHON3_PIP_VERSION).dist-info \
 		$(1)/usr/lib/python$(PYTHON3_VERSION)/site-packages/
-	for _ in \$(seq 1 10) ; do \
-		find $(1)/usr/lib/python$(PYTHON3_VERSION)/site-packages/ -name __pycache__ -exec rm -rf {} \; || continue ; \
-		break ; \
-	done
+	find $(1)/usr/lib/python$(PYTHON3_VERSION)/site-packages/ -path '*/__pycache__/*' -delete
```

Comment:
my only question here, is if this works on BSD/OS X 
I also was thinking of doing it like this initially, but tried to find a portable way of doing it;

i guess, let's have your change and see if this breaks anything

Reply:
It should use the `find` bundled with buildroot (in `staging_dir/host/bin`), which is gnu find, so I think it should be portable.